src/main/java/cart/domain/Product.java

```diff
@@ -17,17 +17,33 @@ public Product(long id, String name, double price, String image) {
 	}
 
 	private void validate(final String name, final double price, final String image) {
-		if (name.isBlank() || name.length() > 20) {
+		if (name.isBlank() || name.isEmpty() || name.length() > 20) {
 			throw new IllegalArgumentException("상품명은 1글자 이상 20글자 이하로 작성해주세요.");
 		}
 		if (price < 0) {
 			throw new IllegalArgumentException("금액은 음수일 수 없습니다.");
 		}
 		if (Objects.isNull(image)) {
```

Comment:
Objects.isNull은 java 1.8 Functional interface와 역사를 같이 하는 친구인데요, 메서드 레퍼런스로 활용하려는 의도가 더 크다고 생각해요. 그래서 저는 보통 아래와 같이 처리합니다 ㅎㅎ
```suggestion
		if (image == null) {
```
<img width="651" alt="image" src="https://user-images.githubusercontent.com/51393021/235447139-2f20c211-93ba-4839-917c-f1d7a7491bc5.png">

(반면에 `Objects.equals(a,b)`는 애용하는데요, a가 null이면 `a.equals(b)`에서 NPE가 발생하기 때문입니다 ㅠ)

Reply:
아하~ 감사합니다.
잊을만하면 찾아오는 NPE. 잊지 않고 고려하며 구현하는게 맞을 것 같네욥ㅎㅎ